### PR TITLE
Add reference to IP handling doc

### LINKED ISSUE #194: Ice Candidate Policy should reference ip-handling draft
https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-12#section-3.4.3 talks about browser local policy. It should make this more concrete by referring to https://tools.ietf.org/html/draft-shieh-rtcweb-ip-handling-00 as the source of this policy.

### PR DESCRIPTION
**Pull request by juberti**
Fixes #194

### CODE REVIEW

**Comment by fluffy** [comment]
Not sure if this is done but what is there LGTM

**Comment by juberti** [comment]
Yes, I think this is complete.

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -766,9 +767,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 even global level.</t>
           </list>
           </t>
-          <t>Although it can be overridden by local policy, the default ICE
-          candidate policy MUST be set to allow all candidates, as this
-          minimizes use of application STUN/TURN server resources.</t>
+          <t>The default ICE candidate policy MUST be set to allow all
+          candidates, as this is generally the desired policy, and also
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: suggest "all types of candidates" because if you apply the draft-shieh rules, then it's not all.

**Comment by ekr** [comment]
LGTM

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: We use "all candidates" elsewhere, so I think this tweak would be insufficient. But perhaps I could speak to the exact meaning of "all candidates" on line 754, and allude to the just-mentioned browser filtering.

**Pull request merged**